level0/provider.tf

```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  region = "us-west-1"
+}
```

Comment:
could you please, add newline at the end of file.

Reply:
done - please review